recipe/conda_build_config.yaml

```diff
@@ -126,6 +128,8 @@ docker_image:                                   # [os.environ.get("BUILD_PLATFOR
 
   - quay.io/condaforge/linux-anvil-cuda:10.2       # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64" and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos6"]
   - quay.io/condaforge/linux-anvil-cos7-cuda:10.2  # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64" and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+  - quay.io/condaforge/linux-anvil-ppc64le-cuda:10.2  # [linux and ppc64le and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+  - quay.io/condaforge/linux-anvil-aarch64-cuda:10.2  # [linux and aarch64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
```

Comment:
We do not support `CUDA 10.2` for `aarch64`. Should we use `quay.io/condaforge/linux-anvil-aarch64-cuda:11.0` instead?

Reply:
Yeah this is trickier than it seems as all the other architectures are using CUDA 10.2. So we would need to drop CUDA 10.2 in order to change to CUDA 11.0. Should add I'm ok with doing that and have checked with others to make sure it won't be too disruptive, but there are some other challenges. Basically we would want to get PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 ) in, which has some blockers atm that have been difficult to address around aligning compiler versions